pkg/apis/monitoring/v1/prometheus_types.go

```diff
@@ -1975,4 +1975,7 @@ type ScrapeClass struct {
 	//
 	// +optional
 	MetricRelabelings []RelabelConfig `json:"metricRelabelings,omitempty"`
+
+	// +optional
```

Comment:
Worth we point to the doc about AttachedMetadata as we're doing with relabel configs